src/Microsoft.ComponentDetection.Contracts/BcdeModels/ScannedComponent.cs

```diff
@@ -15,6 +16,9 @@ public class ScannedComponent
 
         public bool? IsDevelopmentDependency { get; set; }
 
+        [JsonConverter(typeof(StringEnumConverter))]
```

Comment:
Does this need the `JsonConverter` line anymore?

Reply:
without this, 
the enum serialization will be defaulted to int. (for e.g. "dependencyScope" : "0")
This is necessary to get the String value of enum while serializing (in bcde manifest file).